pontoon/checks/management/commands/run_checks.py

```diff
@@ -0,0 +1,43 @@
+import logging
+
+from celery import group
+from django.core.management.base import BaseCommand
+from django.db.models import Q
+
+from pontoon.base.models import Translation
+from pontoon.checks import SUPPORTED_FILES
+from pontoon.checks.tasks import bulk_run_checks
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = 'Run checks on all translations'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--batch-size',
+            action='store',
+            dest='batch_size',
+            default=10000,
+            help='Number of translations to check in a single batch/Celery task',
+        )
+
+    def handle(self, *args, **options):
+        translations_pks = (
+            Translation.objects
+            .filter(
+                entity__resource__format__in=SUPPORTED_FILES
```

Comment:
I think we should also exclude obsolete entities and disabled projects:
```
entity__obsolete=False,
entity__resource__project__disabled=True,
```

You could add `--with-obsolete-entities` and `--with-disabled-projects` as command arguments, but I'm not sure we'd ever need them.